_pages/API/01-general.md

```diff
@@ -68,122 +68,85 @@ This is the interface provided directly by PiWeb Server. It consists of two main
+Additionally it is also possible to specify attributes for a catalog entry which is an *catalog attribute*.
 
-+ *Lower/Upper control limit: K8012/K8013*<br>
-A limit that determines when actions regarding production should be taken based on the number of bad parts
+The unique key of an attribute is important, as it has the purpose of an identifier for PiWeb attributes. The time/date attribute as an example has the key K4. PiWeb already knows a lot of important attributes and their keys by default, which is why changing them is not advised.
 
-+ *Lower/Upper warning limit: K8014/K8015*<br>
-A limit that determines when a warning about bad values should be triggered
+Here are some examples for common attributes known by default:
 
-#### Catalogs
-Catalogs represent a collection of different values. An entry has a number and can have several other attributes. Catalogs are used to simplify decisions at certain points by specifying the available values. The user does not have to enter data by hand, and cannot cause unintended typing errors.
+- *Measured value: K1*<br>
+  The attribute of a measured value, which contains the actual value.
+- *Time/Date: K4*<br>
+  The date of measurement.
+- *Target value: K2100*<br>
+  The desired value a characteristic should have, to which the actual measured value should be as close as possible
+- *Lower/Upper limit: K2110/K2111*<br>
+  The limit of acceptable tolerance between the target value and actual values.
+- *Lower/Upper control limit: K8012/K8013*<br>
+  A limit that determines when actions regarding production should be taken based on the number of bad parts
+- *Lower/Upper warning limit: K8014/K8015*<br>
+  A limit that determines when a warning about bad values should be triggered
 
- An example would be the machine catalog. This contains the different machines which could be used to measure a part. Additional information for each machine can be noted there too. Now an inspector selects the responsible machine from the catalog when entering a measurement, which makes things a lot easier. Other values than those stored in the catalog are not possible. Also simple variants, like the direction catalog, which contains directions like left and right, are possible.
-<hr>
-### Additional Data
-Additional data are attachments that can be added to any entity in the inspection plan. This can be text, images, log files, CAD models or any binary file. There is no limit to the number of files and you can edit or remove additional data as desired.
+### Configuration
+In the PiWeb configuration attributes can be defined ready to be used at parts, characteristics, measurements, measured values and catalog entries. The available catalogs and their contained catalog entries are specified as well.<br>
+<br>
+<img src="/PiWeb-Api/images/configuration.png" class="img-responsive center-block">
```

Comment:
Event though the catalog is shown in the image, the description was removed completely. Is that intentional? I think there should at least be a short description on what a catalog actually is and how it is used.

Reply:
Removing the description was done by mistake. I've re-added the text. Well spotted!